scripts/install.sh

```diff
@@ -71,7 +71,7 @@ delete_tempdir() {
 
 linux_shell() {
   user="$(whoami)"
-  grep "$user" < /etc/passwd | cut -f 7 -d ":" | head -1
+  grep -E "^$user" < /etc/passwd | cut -f 7 -d ":" | head -1
```

Comment:
Same question here on regex special chars. Additionally, should this be `"^$user:" to make sure we don't match a user who is a substring of another user?

Reply:
Whoops, I definitely intended to include the colon in this change. Good catch.